cob_description/urdf/sensors/prosilica.gazebo.xacro

```diff
@@ -25,12 +25,12 @@
         </camera>
         <plugin name="${name}_controller" filename="libgazebo_ros_camera.so">
           <updateRate>10.0</updateRate>
-          <cameraName>${name}_sensor</cameraName>
-          <frameName>/${name}_link</frameName>
+          <cameraName>${name}</cameraName>
+          <frameName>${name}_link</frameName>
           <topicName>${ros_topic}_image_data</topicName>
-          <imageTopicName>/${ros_topic}/image_raw</imageTopicName>
-          <cameraInfoTopicName>/${ros_topic}/camera_info</cameraInfoTopicName>
-          <pollServiceName>/${name}/request_image</pollServiceName>
+          <imageTopicName>${ros_topic}_image_raw</imageTopicName>
+          <cameraInfoTopicName>${ros_topic}_camera_info</cameraInfoTopicName>
```

Comment:
Absolutely correct. 
By considering the discussion in [cob_robots](https://github.com/ipa320/cob_robots/pull/331) about the topic names, I'd suggest to rename the parameter `ros_topic` as something like `orientation` in order to get e.g. `/robot_1/head_color_camera/left|right/image_raw`.